pkg/skaffold/kubernetes/manifest/affinity.go

```diff
@@ -45,21 +45,21 @@ const (
 	nodeArchitectureLabel    = "kubernetes.io/arch"
 )
 
-type ResourceSelectorAffinity struct {
+type ResourceSelectorPodSpec struct {
```

Comment:
Because I'm reusing the same resource selector for `tolerations` as for `affinity`, I realise that it's better named as `ResourceSelectorPodSpec` than `ResourceSelectorAffinity`